ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -336,6 +344,10 @@ struct ZipBuilder {
         "installed: \(installedPods)")
     }
 
+    guard let carthageCoreDiagnosticsXcframework = carthageCoreDiagnosticsXcframeworkFirebase else {
```

Comment:
Interesting that Xcode 12.5 allows these two variables to use the same name, but not 12.2